configuration/postgres/postgres.go

```diff
@@ -126,35 +130,34 @@ func (p *ConfigurationStore) Get(ctx context.Context, req *configuration.GetRequ
 	query, params, err := buildQuery(req, p.metadata.configTable)
 	if err != nil {
 		p.logger.Error(err)
-		return nil, fmt.Errorf("error in configuration store query: '%s' ", err)
+		return nil, fmt.Errorf("error in configuration store query: '%w' ", err)
 	}
 	rows, err := p.client.Query(ctx, query, params...)
 	if err != nil {
 		// If no rows exist, return an empty response, otherwise return the error.
 		if err == sql.ErrNoRows {
 			return &configuration.GetResponse{}, nil
 		}
-		return nil, fmt.Errorf("error in querying configuration store: '%s'", err)
+		return nil, fmt.Errorf("error in querying configuration store: '%w'", err)
 	}
-	items := make(map[string]*configuration.Item)
-	for i := 0; rows.Next(); i++ {
-		var item configuration.Item
-		var key string
-		var metadata []byte
-		v := make(map[string]string)
-		if err := rows.Scan(&key, &item.Value, &item.Version, &metadata); err != nil {
-			return nil, fmt.Errorf("error in reading data from configuration store: '%s'", err)
-		}
-		if err := json.Unmarshal(metadata, &v); err != nil {
```

Comment:
I see this was removed, we don't need to unmarshall the JSON anymore? Please confirm

Reply:
yup. a new struct is added pgNotify 

https://github.com/dapr/components-contrib/blob/96fdc8a99701b308b214fd5dc0c3d4373ae2acfd/configuration/postgres/postgres.go#L53-L56

We are using a new pgxv5 generic API - pgx.CollectRows which simplifies reading the response from pgxpool.Pool.Query 
https://pkg.go.dev/github.com/jackc/pgx/v5#CollectRows